index.html

```diff
@@ -674,6 +674,14 @@ <h2>Typography</h2>
 			<ts-cover id="my-cover"></ts-cover>
 		</ts-root>
 		<script>
+			function onOpen() {
```

Comment:
We need to come up with a better and more consistent solution to show the functionality in future 🤔

Reply:
We can use Open and Close as you refactored before.